sevntu-checks/pom.xml

```diff
@@ -45,6 +45,17 @@
       <scope>test</scope>
       <version>${checkstyle.eclipse-cs.version}</version>
     </dependency>
+    <!-- required for package/jsr305  -->
```

Comment:
Please put name of Check here, as package is changed

Reply:
Done with next commit.